ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge1.cfg

```diff
@@ -119,41 +119,40 @@ router adaptive-virtual-topology
       avt profile PROD-AVT-POLICY-VIDEO id 4
 !
 router internet-exit
-    !
-    exit-group DIRECT-EXIT-POLICY-1
-        local connection IE-Ethernet3
-    !
-    exit-group ZSCALER-EXIT-POLICY-1_PRI
-        local connection IE-Tunnel100
-    !
-    exit-group ZSCALER-EXIT-POLICY-1_SEC
-        local connection IE-Tunnel101
-    !
-    exit-group ZSCALER-EXIT-POLICY-1_TER
-        local connection IE-Tunnel102
-    !
-    exit-group ZSCALER-EXIT-POLICY-2_PRI
-        local connection IE-Tunnel110
-    !
-    exit-group ZSCALER-EXIT-POLICY-2_SEC
-        local connection IE-Tunnel111
-    !
-    exit-group ZSCALER-EXIT-POLICY-2_TER
-        local connection IE-Tunnel112
-    !
-    policy DIRECT-EXIT-POLICY-1
-        exit-group DIRECT-EXIT-POLICY-1
-    !
-    policy ZSCALER-EXIT-POLICY-1
-        exit-group ZSCALER-EXIT-POLICY-1_PRI
-        exit-group ZSCALER-EXIT-POLICY-1_SEC
-        exit-group ZSCALER-EXIT-POLICY-1_TER
-        exit-group system-default-exit-group
-    !
-    policy ZSCALER-EXIT-POLICY-2
-        exit-group ZSCALER-EXIT-POLICY-2_PRI
-        exit-group ZSCALER-EXIT-POLICY-2_SEC
-        exit-group ZSCALER-EXIT-POLICY-2_TER
+   exit-group DIRECT-EXIT-POLICY-1
+      local connection IE-Ethernet3
+   !
+   exit-group ZSCALER-EXIT-POLICY-1_PRI
+      local connection IE-Tunnel100
+   !
+   exit-group ZSCALER-EXIT-POLICY-1_SEC
+      local connection IE-Tunnel101
+   !
+   exit-group ZSCALER-EXIT-POLICY-1_TER
+      local connection IE-Tunnel102
+   !
+   exit-group ZSCALER-EXIT-POLICY-2_PRI
+      local connection IE-Tunnel110
+   !
+   exit-group ZSCALER-EXIT-POLICY-2_SEC
+      local connection IE-Tunnel111
+   !
+   exit-group ZSCALER-EXIT-POLICY-2_TER
+      local connection IE-Tunnel112
+   !
+   policy DIRECT-EXIT-POLICY-1
+       exit-group DIRECT-EXIT-POLICY-1
+   !
+   policy ZSCALER-EXIT-POLICY-1
+       exit-group ZSCALER-EXIT-POLICY-1_PRI
```

Comment:
The indentation here is wrong. (4 spaces instead 3).

Reply:
fixed